tests/installed/nondestructive/itest-bare-unit.sh

```diff
@@ -21,3 +21,24 @@ trap _tmpdir_cleanup EXIT
 /usr/libexec/installed-tests/libostree/test-basic.sh
 /usr/libexec/installed-tests/libostree/test-basic-c
 date
+
+# Test error message when opening a non-world-readable object
+# https://github.com/ostreedev/ostree/issues/1562
+rm repo -rf
+chmod a+rx .
+ostree --repo=repo init --mode=bare
+mkdir files
+touch files/unreadable
+chmod 0 files/unreadable
+ostree --repo=repo commit -b testbranch --tree=dir=files
+# We should be able to read as non-root due to CAP_DAC_OVERRIDE
+ostree --repo=repo ls testbranch >/dev/null
+cat >upriv.sh <<EOF
+#!/bin/bash
+set -xeuo pipefail
+ostree --repo=testclone
+EOF
+if setpriv --reuid bin --regid bin --clear-groups ostree --repo=repo cat testbranch /unreadable 2>err.txt; then
```

Comment:
I'd be more convinced if we did the exact same `cat testbranch /unreadable` test also for root above. :)